File: p10.py
def is_prime(n):
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True
s = 2
for n in range(3, 2000000, 2):
    if is_prime(n):
        s += n
print(s)

File: p10.c
#include <stdio.h>
unsigned int is_prime(unsigned int n)
{
  unsigned int i;
  i = 2;
  while (i * i < n + 1) {
    if (n % i == 0)
      return 0;
    i++;
  }
  return 1;
}
int main()
{
  unsigned int n;
  unsigned long s;
  n = 3;
  s = 2;
  while (n < 2000000) {
    if (is_prime(n))
      s += n;
    n += 2;
  }
  printf("%lu\n", s);
}

